django/README.md

```diff
@@ -1 +1,18 @@
-# Backend
\ No newline at end of file
+# Backend
+
+## Database Migrations
+Create migration
+```bash
+# Log in to the api docker
+docker exec -t -i cthub_api_1 bash
```

Comment:
I'd do docker-compose exec api bash instead. The container name can change, so I'd rather use the service name

Reply:
Good idea.